common/src/main/java/com/microsoft/identity/common/internal/commands/RemoveCurrentAccountCommand.java

```diff
@@ -54,7 +56,23 @@ public RemoveCurrentAccountCommand(@NonNull RemoveAccountCommandParameters param
 
     @Override
     public Boolean execute() throws Exception {
-        return getDefaultController().removeCurrentAccount((RemoveAccountCommandParameters) getParameters());
+        final String methodName = ":execute";
+
+        for (final BaseController controller : getControllers()) {
+            com.microsoft.identity.common.internal.logging.Logger.verbose(
+                    TAG + methodName,
+                    "Executing with controller: "
+                            + controller.getClass().getSimpleName()
+            );
+
+            final boolean removed = controller.removeCurrentAccount((RemoveAccountCommandParameters) getParameters());
+
+            if (removed) {
+                return true;
```

Comment:
Do we not want to remove it from the rest of them, if it happens to exist there?